packages/multicall/src/multicall.ts

```diff
@@ -1,16 +1,21 @@
 
 import { ethers } from 'ethers'
 import { walletContracts } from '@0xsequence/abi'
-import { RpcMethod, RpcVersion } from './constants'
+import { JsonRpcMethod } from './constants'
 import { BlockTag, eqBlockTag, getRandomInt, parseBlockTag, partition, safeSolve } from './utils'
 import { promisify } from 'util'
-import { JsonRpcRequest, JsonRpcResponseCallback, JsonRpcMiddleware, JsonRpcHandlerFunc } from "@0xsequence/network"
+import { JsonRpcVersion, JsonRpcRequest, JsonRpcResponseCallback, JsonRpcMiddleware, JsonRpcHandlerFunc } from "@0xsequence/network"
 
 
-export type MulticallConf = {
+export type MulticallOptions = {
+  // number of calls to enqueue before calling.
   batchSize: number,
+
+  // number of calls to batch within a time window (in milliseconds). If 0, will disable timeWindow.
   timeWindow: number,
-  contract: string
+
+  // only batch calls for a specific contract.
```

Comment:
This contract is the address of the SequenceUtils contracts, maybe we could rename the variable

Reply:
ok, good to kno! I'll update